service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwin.java

```diff
@@ -146,86 +117,62 @@ private void getTwinOperation(URL url, DeviceTwinDevice device) throws IotHubExc
         }
 
         URL url;
-        if ((device.getModuleId() == null) || device.getModuleId().length() ==0)
+        if ((device.getModuleId() == null) || device.getModuleId().length() == 0)
         {
-            /*
-             **Codes_SRS_DEVICETWIN_25_005: [** The function shall build the URL for this operation by calling getUrlTwin**]**
-             */
             url = this.iotHubConnectionString.getUrlTwin(device.getDeviceId());
         }
         else
         {
-            /*
-             **Codes_SRS_DEVICETWIN_28_002: [** The function shall build the URL for this operation by calling getUrlModuleTwin if moduleId is not null **]**
-             */
             url = this.iotHubConnectionString.getUrlModuleTwin(device.getDeviceId(), device.getModuleId());
         }
 
-        /*
-        **Codes_SRS_DEVICETWIN_25_015: [** The function shall serialize the twin map by calling updateTwin Api on the twin object for the device provided by the user**]**
-         */
         TwinState twinState = new TwinState(device.getTagsMap(), device.getDesiredMap(), null);
         String twinJson = twinState.toJsonElement().toString();
 
-        /*
-        **Codes_SRS_DEVICETWIN_25_016: [** The function shall create a new SAS token **]**
-
-        **Codes_SRS_DEVICETWIN_25_017: [** The function shall create a new HttpRequest with http method as Patch **]**
-
-        **Codes_SRS_DEVICETWIN_25_018: [** The function shall set the following HTTP headers specified in the IotHub DeviceTwin doc.
-                                                1. Key as authorization with value as sastoken
-                                                2. Key as request id with a new string value for every request
-                                                3. Key as User-Agent with value specified by the clientIdentifier and its version
-                                                4. Key as Accept with value as application/json
-                                                5. Key as Content-Type and value as application/json
-                                                6. Key as charset and value as utf-8
-                                                7. Key as If-Match and value as '*'  **]**
-
-        **Codes_SRS_DEVICETWIN_25_019: [** The function shall send the created request and get the response **]**
-
-        **Codes_SRS_DEVICETWIN_25_020: [** The function shall verify the response status and throw proper Exception **]**
-         */
-        Proxy proxy = options.getProxyOptions() != null ? options.getProxyOptions().getProxy() : null;
-        DeviceOperations.request(this.iotHubConnectionString, url, HttpMethod.PATCH, twinJson.getBytes(StandardCharsets.UTF_8), String.valueOf(requestId++),options.getHttpConnectTimeout(), options.getHttpReadTimeout(), proxy);
+        Proxy proxy = options.getProxyOptions() != null
```

Comment:
Do we like the style we use for ternary over in .NET to apply here for readability?

Reply:
I have no objections